.travis.yml

```diff
@@ -39,10 +39,10 @@ env:
     - PIPENV_IGNORE_VIRTUALENVS=1
 
   matrix:
-    - COOKIECUTTER_DATABASE=postgresql COOKIECUTTER_ELASTICSEARCH=elasticsearch6 ES_VERSION=6.6.0 E2E=yes
-    - COOKIECUTTER_DATABASE=mysql COOKIECUTTER_ELASTICSEARCH=elasticsearch6 ES_VERSION=6.6.0 E2E=yes
-    - COOKIECUTTER_DATABASE=postgresql COOKIECUTTER_ELASTICSEARCH=elasticsearch5 ES_VERSION=5.6.4
-    - COOKIECUTTER_DATABASE=mysql COOKIECUTTER_ELASTICSEARCH=elasticsearch6 ES_VERSION=6.6.0 COOKIECUTTER_DATAMODEL=None
+    - COOKIECUTTER_DATABASE=postgresql COOKIECUTTER_ELASTICSEARCH=elasticsearch7 ES_VERSION=7.2.0 E2E=yes
+    - COOKIECUTTER_DATABASE=mysql COOKIECUTTER_ELASTICSEARCH=elasticsearch7 ES_VERSION=7.2.0 E2E=yes
+    - COOKIECUTTER_DATABASE=postgresql COOKIECUTTER_ELASTICSEARCH=elasticsearch6 ES_VERSION=6.6.0
```

Comment:
I think it would be nice with each Invenio release to also try to support the latest ES minor + patch versions we support. I guess that would be v6.8.1 (I'll check the changelog and give it a go for `invenio-search` and friends). @lnielsen WDYT?